Makefile.am

```diff
@@ -41,11 +41,15 @@
 # OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 #
 
+WARN_CFLAGS = -Wall -Wextra -Werror
+WARN_CPPFLAGS = -Wall -Wextra -Werror
```

Comment:
This distinction isn't really meaningful, unless you mistook "CPP" for C++ (in this context, it means "C pre-processor").  A single `WARN_FLAGS = ...` would be fine.

Reply:
If we _did_ start wanting different warning options for C vs C++, we'd run into a big annoyance anyway in that they'll both use CPPFLAGS, so we'd need some way to set that differently for different targets.  But we expect it to all just be GCC (or Clang) anyway, it's not like people are gonna be compiling the C code with Intel and the C++ with MSVC or something, so there's no reason to differentiate.

Reply:
My bad, I mistook CPPFLAGS with CXXFLAGS.